### ISSUE TITLE
Questions for LibriSpeech setting

### ISSUE THREAD

**Issue by ccyang123** [openspeech-team/openspeech#187]
# ❓ Questions & Help

I'm learning how to use LibriSpeech Dataset to train the squeezeformer network. 
After 20 epochs training, the evaluation WER (0.6368)  and CER (0.4251) is still very high and not improved anymore.  (training CER(0.6405), WER (0.4278)) 

These results seem "inconsistent" with the accuracy claimed in the paper. (CER, WER < 0.1)
So, I think there must be something wrong with my setting. 

(1) Did any anyone get the CER/WER accuracies below 0.1 using the code from opeeenspeech? 
(2)  Which tokenizer should I use to get good accuracy?  (libri_subword or libr_character?) 
      I used the libri_subword now.

## Details

(a) The training data and evaluation data setting in the "preprocess.py" is as follows: 
LIBRI_SPEECH_DATASETS = [
    "train-960",
    "dev-clean",
    "dev-other",
    "test-clean",
    "test-other",
]

(b) My training script is as follows: 
python ./openspeech_cli/hydra_train.py  dataset="librispeech"  dataset.dataset_download=False  dataset.dataset_path=$DATASET_PATH  dataset.dataset_path="/dataSSD/"  dataset.manifest_file_path=$MANIFEST_FILE_PATH  dataset.manifest_file_path="/dataSSD/LibriSpeech/libri_subword_manifest.txt"  tokenizer=libri_subword  tokenizer.vocab_path=$VOCAB_FILE_PATH  tokenizer.vocab_path="/dataSSD/LibriSpeech"  model=squeezeformer_lstm audio=fbank  trainer=gpu  lr_scheduler=warmup_reduce_lr_on_plateau  ++trainer.accelerator=dp ++trainer.batch_size=128 ++trainer.num_workers=4 criterion=cross_entropy

(c) The training results after 19 epochs are as follows: 
![圖片](https://user-images.githubusercontent.com/41611304/210165257-f55c3d50-fa82-4d18-92b9-371334ae9dfa.png)
![圖片](https://user-images.githubusercontent.com/41611304/210165267-440376b6-4aac-4e7f-bf2a-9997673c5594.png)

**Comment by LaBlazer** [comment]
I'm having the same issue, seems that most the models are not implemented correctly or something
